src/electionguard/tally.py

```diff
@@ -177,6 +177,20 @@ class PlaintextTally(ElectionObjectBase):
     spoiled_ballots: Dict[BALLOT_ID, Dict[CONTEST_ID, PlaintextTallyContest]]
 
 
+@dataclass
+class PublishedPlaintextTally(ElectionObjectBase):
```

Comment:
These classes were made due to the unique serialization to just remove the other elements of the Tally. This simplifies it to just the items to be published. Drastically simplifying the deserialization as well.